assets-and-images.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: Assets and Images
+permalink: /assets-and-images/
+---
+
+Flutter applications include both code and assets (sometimes called resources).
+Assets can include static data, configuration files, or anything else an
+application needs to function; in particular, assets include icons and other
+images displayed in the UI.
+
+## Specifying assets
+
+The flutter build process uses the `flutter.yaml` file to identify assets
+required by an application, as shown in the [Tutorial](/tutorial/). There are
+two sections that specify assets: the general `assets` section and the
+`material-design-icons` section, which includes any
+[Material Design icons](https://design.google.com/icons/) the application uses.
+Both kinds of assets are included in the Flutter application's _asset bundle_,
+which is a special archive that Flutter applications can read from at runtime.
+
+### Asset variants
+
+The build process supports the notion of asset variants: different versions of
+an asset that might be displayed in different contexts. When an asset's path is
```

Comment:
Again, can we show here (not just tell?). I have a hard time imagining what you mean...